theme/brb/views/lib/default.hbs

```diff
@@ -61,6 +62,11 @@
 </head>
 
 <body class="{{#if inline}}window-inline{{/if}}">
+<dl class="skipLinks" role="navigation" aria-label="skip links">
+	<dd><a href="#main-content">{{ $t "global.skipLinks.mainContentSkipLink" }}</a></dd>
+	<dd><a href="#top-navbar">{{ $t "global.skipLinks.topNavbarSkipLink" }}</a></dd>
+	<dd><a href="#footer">{{ $t "global.skipLinks.footerSkipLink" }}</a></dd>
+</dl>
```

Comment:
I think a skip link for the sidebar might be useful as well..?

Reply:
I think sidebar is the first element you navigation to after the skip links